src/metabase/driver/h2.clj

```diff
@@ -149,7 +154,7 @@
   {:pre [(string? connection-string)]}
   (let [[file & options] (str/split connection-string #";+")
         options          (into {} (for [option options]
-                                    (str/split option #"=")))]
+                                    (str/split option #"=" 2)))]
```

Comment:
previously we assume all options have a value, but if you use admin user then the password is empty, so we need to do this split otherwise the split will fail